### DISCUSSION TITLE
Android development documentation

### DISCUSSION THREAD

**Discussion by tajition** [trustwallet/wallet-core#1943] [Q&A]
Hello,

Is there up to date examples of signing transactions in android kotlin?
I am currently trying to use the test cases but smart chain and litecoin are missing for example. smartchain only has address check and litecoin folder is missing alltogether. 
I have eth sol and algorand working so far but I had to make changes from the test case files as they wouldn't work as is. 

Admittedly I new to android development but the lack of easy to use documentation has made this journey a bit of a struggle.  The android integration guide helped me get started but now that I am trying to make coins work in the app (other than eth btc) and sign txn in them has been a struggle.

**Comment by hewigovens** [comment]
Please refer to latest swift test, smart chain -> just check ethereum tests, litecoin -> check bitcoin
